symengine/functions.cpp

```diff
@@ -2647,6 +2661,29 @@ std::size_t LeviCivita::__hash__() const
     return seed;
 }
 
+RCP<const Basic> LeviCivita::subs(const map_basic_basic &subs_dict) const
+{
+    auto it = subs_dict.find(rcp_from_this());
+    if (it != subs_dict.end())
+        return it->second;
+    vec_basic arg(arg_.size());
+    bool this_rcp = true;
+    auto it2 = arg.begin();
+    for (const auto& it: arg_) {
+        *it2 = ((it)->subs(subs_dict));
+        if (this_rcp) {
```

Comment:
The idea of copying it to new vector was to be able to compare args before and after subs, so that `rcp_from_this()` can be used if applicable. If I replace in `arg` I wont be able to do that